### ISSUE TITLE
about syncbn

### ISSUE THREAD

**Issue by xLuge** [wuhuikai/FastFCN#97]
hi, why use syncbn in training and then use BatchNorm in testing ?
should we freeze syncbn in  testing or finetune ? and how ?

**Comment by wuhuikai** [comment]
syncbn is for large batchsize in training.
just freeze it in testing phase.

**Comment by xLuge** [comment]
so,the way to freeze syncbn is setting norm_layer = BatchNorm in test.py? just like this
![image](https://user-images.githubusercontent.com/37794367/117934230-098ff500-b335-11eb-94ed-b65d98fb947d.png

**Comment by wuhuikai** [comment]
After you call model.eval(), BN is freezed.
